runtime/doc/lua.txt

```diff
@@ -1714,11 +1714,43 @@ vim.region({bufnr}, {pos1}, {pos2}, {regtype}, {inclusive})
         `endcol` is exclusive, and whole lines are returned as
         `{startcol,endcol} = {0,-1}`.
 
-vim.schedule_wrap({cb})                                  *vim.schedule_wrap()*
-    Defers callback `cb` until the Nvim API is safe to call.
+vim.schedule_wrap({callback})                            *vim.schedule_wrap()*
+    Returns a function which calls {callback} via |vim.schedule()|.
+
+    Useful to "wrap" existing functions, ensuring they are only called when
+    the Nvim API is safe to call. The returned function passes all arguments
```

Comment:
This is redundant with the `|vim.schedule()|` docs, which are already linked. Any part of this that isn't covered there should be added to the `|vim.schedule()|` docs.

Reply:
> Returns a function which calls {callback} via |vim.schedule()|.
> 
> The returned function passes all arguments to {callback}.

?

Reply:
```vimdoc
vim.schedule({callback})                                      *vim.schedule()*
    Schedules {callback} to be invoked soon by the main event-loop. Useful to
    avoid |textlock| or other temporary restrictions.

    Parameters:  
      • {callback}  fun()
```

Reply:
Not sure I understand your intent Lewis.

Reply:
Sorry, I misunderstood your comment. Ignore mine.